packages/nextjs/app/builders/0x7429CbD5eD20736645723E972bE60B7F6BF5959c/page.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+
+import { useCallback, useEffect, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { Check, Code, Copy, Cpu, Github, Globe, Linkedin, Twitter } from "lucide-react";
+
+const DevvMichaelProfile = () => {
+  const [scrollY, setScrollY] = useState(0);
+  const [mouseX, setMouseX] = useState(0);
+  const [mouseY, setMouseY] = useState(0);
+  const [copied, setCopied] = useState(false);
+
+  const handleCopy = () => {
+    navigator.clipboard
+      .writeText("0x7429CbD5eD20736645723E972bE60B7F6BF5959c")
+      .then(() => {
+        setCopied(true);
+        setTimeout(() => {
+          setCopied(false);
+        }, 3000); // Reset the state after 3 seconds
+      })
+      .catch(err => {
+        console.error("Failed to copy: ", err);
+      });
+  };
+
+  const handleScroll = useCallback(() => {
+    setScrollY(window.scrollY);
+  }, []);
+
+  const handleMouseMove = useCallback((e: MouseEvent) => {
+    setMouseX(e.clientX);
+    setMouseY(e.clientY);
+  }, []);
+
+  useEffect(() => {
+    // Check if window is defined (client-side)
+    if (typeof window !== "undefined") {
+      window.addEventListener("scroll", handleScroll);
+      window.addEventListener("mousemove", handleMouseMove);
+    }
+
+    return () => {
+      if (typeof window !== "undefined") {
+        window.removeEventListener("scroll", handleScroll);
+        window.removeEventListener("mousemove", handleMouseMove);
+      }
+    };
+  }, [handleScroll, handleMouseMove]);
+
+  const parallaxLayers = [
+    { icon: <Code size={48} />, speed: 0.05, initialPos: { x: "18%", y: "14%" } },
+    { icon: <Cpu size={48} />, speed: 0.1, initialPos: { x: "80%", y: "50%" } },
+    { icon: <Globe size={48} />, speed: 0.15, initialPos: { x: "34%", y: "78%" } },
+  ];
+
+  return (
+    <div className="relative min-h-screen overflow-hidden bg-gradient-to-br from-base-200 to-base-300">
```

Comment:
Your Floating Particles work perfectly in light mode. 
But in dark mode you cannot see it, so you might adapt that.